.github/workflows/source_typeform.yml

```diff
@@ -36,20 +36,6 @@ jobs:
           pip install -r requirements.txt
       - name: Check formatting
         run: make fmt-check
-      - name: Setup CloudQuery
```

Comment:
Couldn't figure out why this is failing (wasn't able to reproduce by running the workflow locally).

We'll be deprecating this way of generating docs soon, so I think it's better to remove it for now